swift/ql/test/query-tests/Security/CWE-094/UnsafeJsEval.qlref

```diff
@@ -1 +1,2 @@
-queries/Security/CWE-094/UnsafeJsEval.ql
+query: queries/Security/CWE-094/UnsafeJsEval.ql
+postprocess: TestUtilities/InlineExpectationsTestQuery.ql
```

Comment:
The `.expected` file for this test will need to be updated / deleted.

Reply:
Why?

Reply:
Doesn't an inline expectations test return empty when the annotations are correct?

Reply:
Yes, it produces an empty `testFailures` relation. However, the original query output should still be there.

Reply:
Ah, OK.  That's not necessarily desirable as it could lead to merge conflicts when two people are working on a test file, that would be resolved automatically in inline expectations.  Though I can see that there's more data in the original query (particularly for path problem queries), and any such merge conflicts should be easy to fix.